### ISSUE TITLE
RFC

### ISSUE THREAD

**Issue by TWINGSISTER** [NuclearPlayer/nuclear-docs#5]
It is unclear if Nuclear is a standalone application or if there are web services running on some server  and the whole thing would stop if this server fails.

**Comment by nukeop** [comment]
What do you mean by that? I host nothing that Nuclear depends on, why would it be unclear?

**Comment by TWINGSISTER** [comment]
More thanks for your second prompt reply! It is unclear TO ME! I do not mean that there should be something disguised.  I simply do not understand the architecture. I took a look to the generated XML for the playlist and  I did not seee the plain URL for a song like in a M3U or sptf  so I could imagine that the player stores songs  to some server. Also I saw on the net some comments about Nuclear capping the bitrate and therefore I thought there was some reason for this that could be uploading. Indeed there is no option to leave songs bitrate uncapped.   Before closing this I think it could be useful to have some idea about what the nuclear-web-services role and I think this could go to [this topic](https://github.com/nukeop/nuclear/issues/1315#top)  and therefore I close this
